kit/azure/activity-log/lib/policy_assignments/policy_assignment_es_deploy_azactivity_log.tmpl.json

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "Microsoft.Authorization/policyAssignments",
+  "apiVersion": "2022-06-01",
+  "name": "Deploy-AzActivity-Log",
+  "location": "${default_location}",
+  "dependsOn": [],
+  "identity": {
+    "type": "SystemAssigned"
+  },
+  "properties": {
+    "description": "Deploys the diagnostic settings for Azure Activity to stream subscriptions audit logs to a Log Analytics workspace to monitor subscription-level events",
+    "displayName": "Configure Azure Activity logs to stream to specified Log Analytics workspace",
+    "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/2465583e-4e78-4c15-b6be-a36cbc7c8b0f",
+    "enforcementMode": "Default",
+    "nonComplianceMessages": [
+      {
+        "message": "Azure Activity logs {enforcementMode} be configured to stream to specified Log Analytics workspace."
+      }
+    ],
+    "parameters": {
+      "logAnalytics": {
+        "value": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/${root_scope_id}-mgmt/providers/Microsoft.OperationalInsights/workspaces/${root_scope_id}-la"
```

Comment:
I'm not sure this encodes the right parameters for the log analytics id? I assume this needs to be the id of the log analytics workspace

It's probably easier to just pass the full id as a custom template variable? also not sure about that subscriptions/000 and how that works... is there some other post processing code that resolves the right subscription id?

Reply:
then iam wondering how enterprise scale it does. i copied out of it. I take a look again

Reply:
Regarding the 00000000-0000-0000-0000-000000000000 Subscription ID in the assignment parameters, this is a default value used when subscription_id_management hasn't been declared. https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/207#issuecomment-975309745 . I will set the old value for it.